solar/relaxed_constants.py

```diff
@@ -0,0 +1,51 @@
+from gpkit.constraints.relax import ConstantsRelaxed
+from gpkit.constraints.bounded import Bounded
+from gpkit import Model
+
+"""
+Methods to precondition an SP so that it solves with a relaxed constants algorithim
+and postcondition an SP to ensure all relax values are 1
+"""
+
+def relaxed_constants(model, include_only=None, exclude=None):
```

Comment:
lol it's looking like I should make this inherit from Model and make it official, if it's showing up in multiple repositories at this point :p